src/utils/ar_data_price_chunk_estimator.ts

```diff
@@ -92,9 +92,13 @@ export class ARDataPriceChunkEstimator extends AbstractARDataPriceAndCapacityEst
 
 		const numberOfChunksToUpload = Math.ceil(byteCount.valueOf() / byteCountOfChunk.valueOf());
 
+		// Every 5th chunk, arweave.net pricing adds 1 winston
+		const mysteriousExtraWinston = W(Math.floor(numberOfChunksToUpload / 5));
```

Comment:
So it was humorous for us to call this "mysterious" in the core PR, and we left it that way for the sake of time, but we should call it what it really is so as not to confuse readers in the future. This extra Winston comes about due to the way that the "miner fee share" is computed, so perhaps call it the "minerFeeShareResidual"l

Reply:
I think I have this repo building with npm Core right now, but forgot to export the new chunk estimator in 1.0.1...  ):

So I think this code will be excised once I can import the chunk estimator. I'll add this var refactor to the `dev` merge for next core release